src/components/Footer.js

```diff
@@ -1,110 +1,31 @@
-                        rel="noopener noreferrer"
-                      >
-                        Admin
-                      </a>
-                    </li>
-                  </ul>
-                </section>
-              </div>
-              <div className="column is-4">
-                <section>
-                  <ul className="menu-list">
-                    <li>
-                      <Link className="navbar-item" to="/blog">
-                        Latest Stories
-                      </Link>
-                    </li>
-                    <li>
-                      <Link className="navbar-item" to="/contact">
-                        Contact
-                      </Link>
-                    </li>
-                  </ul>
-                </section>
-              </div>
-              <div className="column is-4 social">
-                <a title="facebook" href="https://facebook.com">
-                  <img
-                    src={facebook}
-                    alt="Facebook"
-                    style={{ width: '1em', height: '1em' }}
-                  />
+      <footer className="footer has-background-black has-text-black-ter">
+        <div className="content has-text-centered has-background-black has-text-black-ter">
+          <div className="container has-background-black has-text-black-ter footer-content">
+              <div className="is-4 social">
+                <a title="facebook" href="https://www.facebook.com/Pi515/">
+                  <FontAwesomeIcon icon={faFacebookF} color="black" size="lg"/>
                 </a>
-                <a title="twitter" href="https://twitter.com">
-                  <img
-                    className="fas fa-lg"
-                    src={twitter}
-                    alt="Twitter"
-                    style={{ width: '1em', height: '1em' }}
-                  />
+                <a title="youtube" href="https://www.youtube.com/channel/UCDOtl8WYPLelXNDQhzKBTKQ">
+                  <FontAwesomeIcon icon={faYoutube} color="black" size="lg"/>
                 </a>
-                <a title="instagram" href="https://instagram.com">
-                  <img
-                    src={instagram}
-                    alt="Instagram"
-                    style={{ width: '1em', height: '1em' }}
-                  />
+                <a title="twitter" href="https://twitter.com/LOVEPI515">
+                  <FontAwesomeIcon icon={faTwitter} color="black" size="lg"/>
+                </a>
+                <a title="linkedin" href="https://www.linkedin.com/company/pi-515/">
+                  <FontAwesomeIcon icon={faLinkedin} color="black" size="lg"/>
                 </a>
-                <a title="vimeo" href="https://vimeo.com">
-                  <img
-                    src={vimeo}
-                    alt="Vimeo"
-                    style={{ width: '1em', height: '1em' }}
-                  />
+                <a title="instagram" href="https://instagram.com">
```

Comment:
I'm not sure this is needed as they don't have an Instagram as far as I know.

Reply:
They do. She created it last night.